StartupSession/Link/Utils.apln

```diff
@@ -755,8 +757,13 @@
           :Select nc
           :Case 2.1 ⍝ variable
               val←ns⍎name ⍝ what if the var already has a file
-              sub←SubExt val
-              Cut←{lb←⎕UCS ⍺ ⋄ t←lb,⍵ ⋄ (≢⍺)↓¨t⊂⍨lb⍷t}
+              :If 2=opts.⎕NC'noTextFiles'
```

Comment:
Why is the additional test on NC of noTextFiles necessary here, but not in other places?

Reply:
Removed.